src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -1850,15 +1850,15 @@ private void EnsureRootBoundForNullabilityIfNecessary()
             // part.
             var binder = GetEnclosingBinder(GetAdjustedNodePosition(bindableRoot));
             var boundRoot = Bind(binder, bindableRoot, diagnostics);
-            boundRoot = RewriteNullableBoundNodes(boundRoot, binder, diagnostics);
+            boundRoot = RewriteNullableBoundNodesWithCheckpoints(boundRoot, binder, diagnostics, out var checkpointMap);
```

Comment:
>var checkpointMap [](start = 101, length = 17)

`_` or `var snapshotManager`

Reply:
I'll use `_` for now. We'll consume it later.

---
In reply to: [289510893](https://github.com/dotnet/roslyn/pull/35850#discussion_r289510893) [](ancestors = 289510893)